Frontend/src/pages/crowdactions/Detail/SocialMediaSharing.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { EmailIcon, EmailShareButton, FacebookIcon, FacebookShareButton, FacebookShareCount, LinkedinIcon, LinkedinShareButton, TumblrIcon, TumblrShareButton, TumblrShareCount, TwitterIcon, TwitterShareButton, WhatsappIcon, WhatsappShareButton } from 'react-share';
+import { ICrowdaction } from '../../../api/types';
+
+import styles from './SocialMediaSharing.module.scss';
+
+const SocialMediaSharing = ({ crowdaction }: any) => {
+
+  const action = crowdaction as ICrowdaction;  
+  const shareUrl = window.location.href;
+
+  return (
+    <div className={styles.shareButtons}>
+      <FacebookShareButton quote={`${action.name}: ${action.proposal}`} hashtag={action.tags.map(tag => `#${tag.tag.name}`).join(' ')} url={shareUrl}>
+        <FacebookIcon size={32} round />
+      </FacebookShareButton>
+      <FacebookShareCount url={shareUrl}>
+        {count => count}
+      </FacebookShareCount>
+
+      <TwitterShareButton url={shareUrl} title={action.name} hashtags={action.tags.map(tag => `#${tag.tag.name}`)}>
+        <TwitterIcon size={32} round />
+      </TwitterShareButton>
+
+      <LinkedinShareButton url={shareUrl} title={action.name} summary={action.proposal}>
+        <LinkedinIcon size={32} round />
+      </LinkedinShareButton>
+
+      <TumblrShareButton url={shareUrl} title={action.name} caption={action.proposal} tags={action.tags.map(tag => tag.tag.name)}>
+        <TumblrIcon size={32} round />        
+      </TumblrShareButton>
+      <TumblrShareCount url={shareUrl}>
+        {count => count}
+      </TumblrShareCount>
+
```

Comment:
I would've expected instagram here somewhere, giving how much it's used to promote collaction

Reply:
There is no Instagram share button (or API) for websites. At least I couldn't find one.